Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.7-alpine
+WORKDIR /code
+ENV FLASK_APP app.py
+ENV FLASK_RUN_HOST 0.0.0.0
+RUN apk add --no-cache gcc musl-dev linux-headers
```

Comment:
You'll want this higher up in the docker file. Each line in this file is like a layer of an onion on the docker file system. Docker can detect when a layer changes which makes it recreate every layer on top. Due to system packages changes less often you want towards the top.. probably right after the `FROM`

Reply:
I don't think I understand, Which part/line?  It is right after the `FROM`? in the Dockerfile?